### PR TITLE
WIP: new DV plugin

### PR DESCRIPTION
**Pull request by edoardo**
This branch contains the new plugin (React component) as a separate package in the same repo, and changes to the DV app for using it.

All analytics API requests and chart API use is done in the plugin.
The app only consumes the plugin passing the current AO.

App tests are currently broken and need to be reviewed.

### CODE REVIEW

File: packages/app/src/components/Interpretations/Interpretations.js [hunk 1]
```diff
@@ -25,12 +25,12 @@ export class Interpretations extends Component {
             // the other scenario is when the URL changes because of a click inside the Interpretations component
             if (history.location.pathname !== interpretationUrl) {
                 history.push(interpretationUrl);
-            }
 
-            this.props.acSetUiInterpretation({
-                id: interpretation.id,
-                created: interpretation.created,
-            });
+                this.props.acSetUiInterpretation({
```
Comment (jenniferarnesen) [inline packages/app/src/components/Interpretations/Interpretations.js]: Why was this moved inside the `if`? Is this a bug fix?

Comment (edoardo) [reply packages/app/src/components/Interpretations/Interpretations.js]: not a bug fix, but an optimization. see commit description: https://github.com/dhis2/data-visualizer-app/pull/195/commits/32dfdd713f0ae003d51d3062301695fd81c94824

**Comment by edoardo** [comment]
Closing in favour of other 2 PRs, one dedicated to plugin related changes so the plugin can be published before the app changes.
